### ISSUE TITLE
create "Correct" files for Indentation

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#15340]
create correct flles that are fully auto formatted and Checkstyle does not give any violation.

we can copy existing Inputs and auto format, and all violations should be a defect on checksyle side (with manual confirmation in style guide) that formatting is correct.

**Comment by romani** [comment]
can we find in web any project that use auto format and checkstyle in same time ?
if project is just use auto format, we can simply run Indentation module over it and spot conflicts of checkstyle and formatter and consider a way to improve Indentation module to produce less violations.

@rdiachenko , do you know about any of such ?

**Comment by Zopsss** [comment]
List of sections to keep track of issue:

- [ ] 4.1.1 Use of optional braces
- [ ] 4.1.2 Nonempty blocks: K & R style
- [ ] 4.1.3 Empty blocks: may be concise
- [ ] 4.2 Block indentation: +2 spaces
- [ ] 4.3 One statement per line
- [ ] 4.4 Column limit: 100
- [ ] 4.5.1 Where to break   
- [ ] 4.5.2 Indent continuation lines at least +4 spaces
- [ ] 4.6.1 Vertical Whitespace
- [ ] 4.6.2 Horizontal whitespace
- [ ] 4.8.2.1 One variable per declaration
- [ ] 4.8.2.2 Declared when needed
- [ ] 4.8.3.2 No C-style array declarations
- [ ] 4.8.4.1 Indentation
- [ ] 4.8.4.2 Fall-through: commented
- [ ] 4.8.4.3 Presence of the default label
- [ ] 4.8.5.2 Class Annotations
- [ ] 4.8.5.3 Methods And Constructors Annotations
- [ ] 4.8.5.4 Field Annotations
- [ ] 4.8.6.1 Block comment style
- [ ] 4.8.7 Modifiers
- [ ] 4.8.8 Numeric Literals

**Comment by Zopsss** [comment]
Indentation defect found in formatter at: https://github.com/checkstyle/checkstyle/pull/15521#issuecomment-2299323787

Original code:

```java
    switch (a) {
      // comment
      // comment
      // comment
      case 1:
```

```java
    switch (a) {
      // comment
      case 1:
```

Formatted code:

```java
    switch (a) {
        // comment
        // comment
        // comment
      case 1:
```

```java
    switch (a) {
        // comment
      case 1:
```

An issue has been opened for this defect on formatter's GH: https://github.com/google/google-java-format/issues/1147, visit there for more detailed description
